### PR TITLE
#352 DFN notebook

### LINKED ISSUE #352: Notebook DFN

### PR DESCRIPTION
**Pull request by rtimms**
# Description

Added a notebook for the DFN

Fixes #352 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Key checklist:

- [ ] No style issues: `$ flake8`
- [ ] All tests pass: `$ python run-tests.py --unit`
- [ ] The documentation builds: `$ cd docs` and then `$ make clean; make html`

You can run all three at once, using `$ python run-tests.py --quick`.

## Further checks: 

- [ ] Code is commented, particularly in hard-to-understand areas
- [ ] Tests added that prove fix is effective or that feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Scottmar93** [comment]
Fails due to same tolerance issue that @martinjrobins. Should be able to just merge from master now

**Comment by Scottmar93** [comment]
Failing with 10^(-5) maybe we want 10^(-4) as tolerance? 

I'm sure this is good practice

**Comment by martinjrobins** [comment]
Yes, need to investigate python 3.5 more (see #382)

File: examples/notebooks/models/DFN.ipynb [hunk 1]
```diff
@@ -96,21 +236,20 @@
...
+    "|$\\sigma_{\\text{k}}$        | $\\sigma_{\\text{k}}^*R^* T^*/(I^*F^*L^*)$| Dimensionless solid conductivity |\n",
+    "|$\\mathcal{C}_{\\text{k}}$   | $\\tau_{\\text{k}}^*/\\tau_{\\text{d}}^*$   | Ratio of solid diffusion and discharge timescales |\n",
+    "|$\\mathcal{C}_{\\text{e}}$   |$\\tau_{\\text{e}}^*/\\tau_{\\text{d}}^*$    |Ratio of electrolyte transport and discharge timescales|\n",
+    "|$\\mathcal{C}_{\\text{r,k}}$ |$\\tau_{\\text{r,k}}^*/\\tau_{\\text{d}}^*$  |Ratio of reaction and discharge timescales|\n",
+    "|$a_{\\text{k}}$             |$a_{\\text{k}}^* R_{\\text{k}}^*$          | Product of particle radius and surface area density|\n",
+    "|$\\gamma_{\\text{k}}$        |$c_{\\text{k,max}}^*/c_{\\text{n,max}}^*$  |Ratio of maximum lithium concentrations in solid|\n",
+    "|$\\gamma_{\\text{e}}$        |$c_{\\text{e,typ}}^*/c_{\\text{n,max}}^*$  |Ratio of maximum lithium concentration in the negative electrode solid and typical electrolyte concentration|\n",
+    "\n",
+    "Note that the dimensionless parameters $\\epsilon_{\\text{k}}$, $\\text{b}$, and $t^+$ are already provided in the parameter file LCO.csv.  "
```
Comment (Scottmar93) [inline examples/notebooks/models/DFN.ipynb]: The parameter file name has changed now

**Pull request merged**